.changeset/green-pumas-know.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/perseus-core": major
```

Comment:
I don't think this needs to be a major.  We aren't making any backwards-incompatible changes.  I'd mark this as a **patch** give that we aren't changing any production behaviour nor exporting any new functions/types from the `perseus-core` package.